### PR TITLE
document page helpers

### PR DESCRIPTION
**Pull request by jwoertink**
This adds in docs for some of the pages helpers.

### CODE REVIEW

File: src/actions/guides/frontend/rendering_html.cr [hunk 1]
```diff
@@ -263,6 +263,95 @@ class Guides::Frontend::RenderingHtml < GuideAction
...
+
+    Use this to alternate between different values
+
+    ```crystal
+    array = [1, 2, 3]
+    cycle(array) #=> 1
+    cycle(array) #=> 2
+    cycle(array) #=> 3
+    cycle(array) #=> 1
+    ```
```
Comment (paulcsmith) [inline src/actions/guides/frontend/rendering_html.cr]: Let's remove this for now. I'm not sure it is even useful and it seems fairly complex to implements so we might just remove it from Lucky

Comment (jwoertink) [reply src/actions/guides/frontend/rendering_html.cr]: Agreed. That was sort of difficult to write lol

**Comment by jwoertink** [comment]
Turns out this makes the docs a little bit more difficult. For example [this method](https://github.com/luckyframework/lucky/blob/master/src/lucky/page_helpers/text_helpers.cr#L101) contains HTML, but seems to return a String instead of writing to the page. 

I'm wondering if maybe those should just be broken out. Like one module that contains page helpers that return a flat string, and then a separate module of page helpers that write to the page....

In any case, I tried to make it clear in the docs which ones do what, so the wording may be a bit wonky. 

/cc. @edwardloveall

**Comment by paulcsmith** [comment]
Yeah totally. I think there should be HtmlHelpers and TextHelpers so there is a clear file ration between those. Could you open an issue? I think that’d be a good cleanup task to make things nicer

**Comment by edwardloveall** [comment]
Agreed, good distinction. Thanks for doing this @jwoertink !

**Pull request merged**